Wrappers/Python/cil/optimisation/functions/MixedL21Norm.py

```diff
@@ -105,7 +105,14 @@ def proximal(self, x, tau, out=None):
             
         else:
             
-            tmp = (x/tau).pnorm(2)
+            try:
+                x.divide(tau,out=x)
+                tmp = x.pnorm(2)
+                x.multiply(tau,out=x)
+            except TypeError:
+                x_scaled = x.divide(tau)
+                tmp = x_scaled.pnorm(2)
+ 
             tmp_ig = 0.0 * tmp
             (tmp - 1).maximum(0.0, out = tmp_ig)
```

Comment:
Maybe this could also be split and made inline precalculating `tmp -= 1`?

Reply:
`tmp` is reused later, but also this is a small array so not a big copy of the dataset.